doc/source/ray-more-libs/lightgbm-ray.md

```diff
@@ -1,8 +1,572 @@
-<!--
-DO NOT EDIT THIS FILE
+(lightgbm-ray)=
 
```

Comment:
@thetwotravelers we need to revert these changes. same for xgboost below. other than that, outstanding contribution!

Reply:
thanks for catching this!